README.md

```diff
@@ -12,6 +13,17 @@
 
 *All combined notes, links and documents are listed below:
 
+mac-6-add-sp-link
+#### [Github Documentation](https://docs.github.com/en/github)
```

Comment:
I really like the inclusion of the original docs!

Reply:
_Thank you for your help. michaelchrisco!_